draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -454,6 +454,8 @@ To ensure readability and replicability, the examples in this specification do n
 #### Creating an SD-JWT {#creating_sd_jwt}
 
 An SD-JWT is a JWT that MUST be signed using the Issuer's private key.
+It MUST use a JWS asymmetric digital signature algorithm from [@IANA.JWS.Algorithms].
```

Comment:
Are there any asymmetric digital signatures that are not registered in IANA? Are we prohibiting those?

Reply:
That's a good question. There might be unregistered ones (JWS allows for it https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.1) and this text would prohibit their use in SD-JWT. Which is kind of an edge case but probably not what we want. I borrowed that text from DPoP where an AD had suggested it be more restrictive with respect to the registry. But I didn't love it there and your question reminds me of that and that it's probably less appropriate here. I'll rephrase this sentence so as not to be prohibitive.